static_build/scss/components/_day_picker.scss

```diff
@@ -0,0 +1,148 @@
+/* DayPicker styles */
+$day-picker-padding: 10%;
+$calendar-row-height: 40px;
+$calendar-day-size: 30px;
+
+.DayPicker {
```

Comment:
@andela-rsadiq For the calendar widget, I opted to use the popular [react-day-picker](http://react-day-picker.js.org/) component. To apply our custom styles to the component, the [styling section in the docs](http://react-day-picker.js.org/Styling.html) mandates that we use these predefined classes so as much as we'd love to define our convention, we have to stick with theirs for that component.

Their class naming convention loosely follow a widely used CSS methodology called BEM. You can read more about it [here](https://css-tricks.com/bem-101/)